05_run_ocp.sh

```diff
@@ -106,7 +108,23 @@ ssh -o "StrictHostKeyChecking=no" core@$IP sudo modprobe iscsi_tcp
 # doesn't exist on the host
 ssh -o "StrictHostKeyChecking=no" core@$IP sudo mkdir /run/httpd
 
+# Internal dnsmasq should reserve IP addresses for each master
+cp -rf ironic/dnsmasq.conf ~stack/dnsmasq.conf
```

Comment:
This doesn't work if you're not running this as the stack user - also doesn't need the recursive switch.

Also if we're going to copy to a working dir, can we make it a variable, as this will break when https://github.com/metalkube/dev-scripts/pull/39 gets fixed up I expect?

Reply:
Right, `/tmp` should be used here, so that original file won't be rewritten

Reply:
Fixed